src/main/java/ee/carlrobert/codegpt/completions/ollama/DownloadOllamaModelAction.java

```diff
@@ -48,7 +47,8 @@ public DownloadModelAction(
 
   @Override
   public void actionPerformed(@NotNull AnActionEvent e) {
-    ProgressManager.getInstance().run(new DownloadBackgroundTask(e.getProject()));
+    // todo use ollama pull cmd
```

Comment:
In theory, you can download a Model for a remote server to, with [ollama/pull-a-model](https://github.com/ollama/ollama/blob/main/docs/api.md#pull-a-model)